### DISCUSSION TITLE
Mysql and Keycloak start but not Stig Manager

### DISCUSSION THREAD

**Discussion by gramenis** [NUWCDIVNPT/stig-manager#860] [Q&A]
I have installed Docker CE and pulled the stig manager, keycloak, and mysql images. I have also created the docker-compose.yaml file per the instructions.  When I do docker compose up in the directory where the docker-compose file is I see that mysql is started, keycloak is started but an error when starting stig manager. 

Error:  Attaching to docker-api-1, docker-auth-1, docker-db-1
Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: waiting for init preliminary setup: read init-p: connection reset by peer  unknown

I was able to bring up the Keycloak admin and login as well as verified mysql 8.  Running docker ps -a I see
3 images with status created.  I also tried starting up the images individually but when I got to stig-manager I got same error.

Update:  After trying to start up the containers individually the error I see is 404 Not Found when trying to access http://IP of Server:8080/auth/realms/stigman  ; this retries several times then errors out

Update #2:  I have success!!  Have to document what I did but in short I had to use http://IP of Docker Server:8080/realms/stigman, DB Port of 50001, and I had to put the STIGMAN_DB_PASSWORD when running it manually.

**Comment by cd-rite** [comment]
Just noticed this, but glad you got it running!!
